src/theory/quantifiers/sygus/sygus_pbe.h

```diff
@@ -636,31 +680,40 @@ class CegConjecturePbe : public SygusModule
   //------------------------------ end strategy registration
 
   //------------------------------ constructing solutions
-  class UnifContext {
-  public:
-   UnifContext() : d_has_string_pos(role_invalid) {}
-   /** this intiializes this context for function-to-synthesize c */
-   void initialize(CegConjecturePbe* pbe, Node c);
+  /** Unification context
+   *
+   * This class maintains state information during calls to
+   * CegConjecturePbe::constructSolution, which implements unification-based
+   * approaches for construction solutions to synthesis conjectures.
```

Comment:
add a reference to Alur et al. CAV 2015?

Reply:
Its the TACAS 2017 paper, plus extensions. I already cite this in the header.